docs/release-notes/NuGet-6.13.md

```diff
@@ -23,6 +17,102 @@ NuGet distribution vehicles:
 
 <sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
 
-## Not yet released
+## Summary: What's New in 6.13
```

Comment:
I just relaised there are 6.13.0 and 6.13.1 releases going out.  I think we should separate them, tell customers which versions of VS & .NET SDK they ship in, and what the differences are between the two.

Reply:
Yes, this is what we do usually.